src/octoprint/plugins/softwareupdate/__init__.py

```diff
@@ -225,20 +226,20 @@ def get_settings_defaults(self):
 		return {
 			"checks": {
 				"octoprint": {
-					"type": "github_release",
-					"user": "foosel",
+					"user": "mrbeam",
 					"repo": "OctoPrint",
 					"method": "pip",
-					"pip": "https://github.com/foosel/OctoPrint/archive/{target_version}.zip",
+					"pip": "https://github.com/mrbeam/OctoPrint/archive/{target_version}.zip",
 					"update_script": default_update_script,
 					"restart": "octoprint",
-					"stable_branch": dict(branch="master", commitish=["master"], name="Stable"),
-					"prerelease_branches": [dict(branch="rc/maintenance",
-					                             commitish=["rc/maintenance"],             # maintenance RCs
-					                             name="Maintenance RCs"),
-					                        dict(branch="rc/devel",
-					                             commitish=["rc/maintenance", "rc/devel"], # devel & maintenance RCs
-					                             name="Devel RCs")]
+					"stable_branch": dict(branch="mrbeam2-stable", commitish=["mrbeam2-stable"], name="Stable"),
+					"prerelease_branches": [dict(branch="mrbeam2-alpha",
+					                             commitish=["mrbeam2-stable", "mrbeam2-alpha", "mrbeam2-beta"],
+					                             name="Alpha"),
+					                        dict(branch="mrbeam2-beta",
+					                             commitish=["mrbeam2-stable", "mrbeam2-beta"],
+					                             name="Beta")
+											],
```

Comment:
Why not add `develop` here?

Reply:
then we always have to make prereleases for develop as well, I was thinking about this at first too. But I think it makes more sense to use the github_commit for the develop tier